scripts/run_backend_tests.py

```diff
@@ -603,7 +603,43 @@ def main(args: Optional[List[str]] = None) -> None:
     if parsed_args.generate_coverage_report:
```

Comment:
How do the coverage issues raised here compare to those reflected in the `incomplete_coverage` variable above? When `incomplete_coverage != 0`, we raise an exception and won't ever reach the code you added.

Reply:
@U8NWXD In this specific line, we retrieve the incomplete_coverage value from the following line of code:
`check_test_results(tasks, task_to_taskspec, parsed_args.generate_coverage_report)`

Within the check_test_results function, we increase the incomplete_coverage under two conditions:
`spec.test_target not in coverage_exclusions and float(coverage) != 100.0`

when we see the 4th backend test shard the **spec.test_target** value is "**core.jobs.batch_jobs.subtopic_migration_jobs_test,**" which is not listed in the **coverage_exclusions**. However, the coverage for this particular file is **100%**. As a result, the **incomplete_coverage** value is **0**.

The same situation is occurring in the other four shards as well, leading to an **incomplete_coverage** value of **0** for all four shards.

Reply:
Just a note -- let's ignore (1) for the purposes of making the coverage log clearer. We should focus on (2). I've had a chat with @U8NWXD about removing the functionality for (1) since we are not planning to enforce it.